.coveragerc

```diff
@@ -514,6 +514,10 @@ omit =
     homeassistant/components/mychevy/*
     homeassistant/components/mycroft/*
     homeassistant/components/mycroft/notify.py
+    homeassistant/components/myio/sensor.py
+    homeassistant/components/myio/__init__.py
+    homeassistant/components/myio/const.py
+    homeassistant/components/myio/config_flow.py
```

Comment:
The configuration flow should not be excluded from test coverage. We require configuration flows to implement tests with 100% coverage.

Reply:
config_flow deleted from .coveragerc